uber/models.py

```diff
@@ -904,7 +904,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     last_name     = Column(UnicodeText)
     email         = Column(UnicodeText)
     birthdate     = Column(Date, nullable=True, default=None)
-    age_group     = Column(Choice(c.AGE_GROUPS), default=c.AGE_UNKNOWN, nullable=True)
+    age_group     = Column(Choice(c.AGE_GROUP_OPTS), default=c.AGE_UNKNOWN, nullable=True)
```

Comment:
Dangit.

I mean it HAS to do something, I tested it repeatedly and the behavior is
definitely different between the list vs the dict.

On Fri, Sep 11, 2015, 08:36 Eli Courtwright <notifications@github.com>
wrote:

> In uber/models.py
> <https://github.com/magfest/ubersystem/pull/1493#discussion_r39266453>:
>
> > @@ -904,7 +904,7 @@ class Attendee(MagModel, TakesPaymentMixin):
> >      last_name     = Column(UnicodeText)
> >      email         = Column(UnicodeText)
> >      birthdate     = Column(Date, nullable=True, default=None)
> > -    age_group     = Column(Choice(c.AGE_GROUPS), default=c.AGE_UNKNOWN, nullable=True)
> > +    age_group     = Column(Choice(c.AGE_GROUP_OPTS), default=c.AGE_UNKNOWN, nullable=True)
>
> I'm roughly 99.9% positive that this change doesn't do anything. The
> Choice class accepts either a list or dictionary; if you look at
> https://github.com/magfest/ubersystem/blob/master/uber/models.py#L104
> then you can see that it just immediately converts whatever is passed to a
> dict anyway, so this shouldn't matter.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1493/files#r39266453>.
>